### ISSUE TITLE
Data Scientist Guidelines for Recsys models - Example notebook series

### ISSUE THREAD

**Issue by EvenOldridge** [NVIDIA-Merlin/models#407]
- [ ] Dealing with overfitting
 - [ ] Stabilizing training (Batchnorm and layernorm)
 - [ ] Learning rate scheduling
